setup.py

```diff
@@ -40,7 +40,7 @@ def local_file(name):
         'Topic :: System :: Monitoring',
     ],
     install_requires=[
-        'attrs==20.3.0',
```

Comment:
Could we instead set a minimum supported version?

Reply:
For me, setting a minimum version rather than requiring a specified version would be better. Specifying an individual version (even if kept up to date) does suggest some sort of dark reliance on internal implementation details in attr.

Reply:
What minimum version do you require? Please could you add this as the minimum in your PR?

Reply:
For my particular use case, 15.2.0 would be fine, and a version I have tested.